cdk/lib/admin-console-dynamo.ts

```diff
@@ -0,0 +1,46 @@
+import type { GuStackProps } from '@guardian/cdk/lib/constructs/core';
+import { GuStack } from '@guardian/cdk/lib/constructs/core';
+import type { App, CfnElement } from 'aws-cdk-lib';
+import { RemovalPolicy } from 'aws-cdk-lib';
+import { AttributeType, ProjectionType, Table } from 'aws-cdk-lib/aws-dynamodb';
+
+export class AdminConsoleDynamo extends GuStack {
+  constructor(scope: App, id: string, props: GuStackProps) {
+    super(scope, id, props);
+
+    const buildTable = (channel: string) => {
+      const id = `${channel[0].toUpperCase() + channel.slice(1)}TestsDynamoTable`;
+
+      const table = new Table(this, id, {
+        tableName: `${channel}-tests-${this.stage}`,
+        removalPolicy: RemovalPolicy.RETAIN,
+        readCapacity: 4,
+        writeCapacity: 4,
+        partitionKey: {
+          name: 'name',
```

Comment:
this makes the test name the primary key - we already require each test to have a unique name